source/common/http/filter/lua/wrappers.h

```diff
@@ -2,6 +2,7 @@
 
 #include "envoy/http/header_map.h"
 
+#include "common/config/well_known_names.h"
```

Comment:
is this needed here?

Reply:
Yes it is not needed. Thank you. Added here: 6319e08